Error when using feedback together with Gitlab

Hi, I tried to use this pluging together with the Gitlab plugin. It gave an error:

<details>

```
$ repobee issues feedback -a ha-0 --multi-issues-file grading_feedback.md --sf canvas-students-file-HA0.lst --tb
Processing issue "Grading Specification" for 209929-ha-0: Documentation: 15
Failure Report doctest: 15
Failure Report pytest: 15
Create Issue: 5
Bugs: 20
Success Report doctest: 12.5
Success Report pytest: 12.5
Close Issue: 5
Total: 100
Grade: 10
Open issue "Grading Specification" in repo 209929-ha-0? (y/n) y
[ERROR] a TypeError occured unexpectedly: sequence item 1: expected str instance, StudentTeam found
[ERROR] Critical exception
Traceback (most recent call last):
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/_repobee/ext/gitlab.py", line 80, in _try_api_request
    yield
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/_repobee/ext/gitlab.py", line 228, in get_repo
    project = self._gitlab.projects.get("/".join(path))
TypeError: sequence item 1: expected str instance, StudentTeam found

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/_repobee/main.py", line 174, in _main
    _repobee.cli.dispatch.dispatch_command(
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/_repobee/cli/dispatch.py", line 46, in dispatch_command
    ext_cmd.command(api=api)
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/repobee_feedback/feedback.py", line 108, in command
    callback(self.args, api)
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/repobee_feedback/feedback.py", line 49, in callback
    repo = api.get_repo(repo_name, repo_name_to_team[repo_name])
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/_repobee/ext/gitlab.py", line 229, in get_repo
    return self._wrap_project(project)
  File "/usr/lib/python3.8/contextlib.py", line 131, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/lars/.repobee/env/lib/python3.8/site-packages/_repobee/ext/gitlab.py", line 97, in _try_api_request
    raise plug.UnexpectedException(
repobee_plug.exceptions.UnexpectedException: a TypeError occured unexpectedly: sequence item 1: expected str instance, StudentTeam found
[ERROR] A plugin exited with an error
```

</details>

I have changed https://github.com/repobee/repobee-feedback/blob/9f0e35624777652e1bae3560a7f005310deca55d/repobee_feedback/feedback.py#L49 to:
```python
repo = api.get_repo(repo_name, repo_name_to_team[repo_name].name)
```

And now it works again.

--- a/repobee_feedback/feedback.py
+++ b/repobee_feedback/feedback.py
@@ -11,7 +11,7 @@ import pathlib
 import re
 import sys
 import argparse
-from typing import Iterable, Tuple, List
+from typing import Iterable, Tuple, List, Mapping
 
 import repobee_plug as plug
 
@@ -21,7 +21,7 @@ BEGIN_ISSUE_PATTERN = r"#ISSUE#(.*?)#(.*)"
 
 
 def callback(args: argparse.Namespace, api: plug.PlatformAPI) -> None:
-    repo_name_to_team = {
+    repo_name_to_team: Mapping[str, plug.StudentTeam] = {
         plug.generate_repo_name(
             student_team.name, assignment_name
         ): student_team
@@ -45,7 +45,7 @@ def callback(args: argparse.Namespace, api: plug.PlatformAPI) -> None:
             issue, repo_name, args.truncation_length
         )
         if open_issue:
-            repo = api.get_repo(repo_name, repo_name_to_team[repo_name])
+            repo = api.get_repo(repo_name, repo_name_to_team[repo_name].name)
             api.create_issue(issue.title, issue.body, repo)
         else:
             plug.echo("Skipping {}".format(repo_name))

